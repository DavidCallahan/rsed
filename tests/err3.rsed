skip to required "/asx/"
required "/asxxxxx/"
replace "/TARGETS" with "/BUCK" all


