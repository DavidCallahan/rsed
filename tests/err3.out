input 9: script 2: failed required pattern: 
